Formula/c/chainloop-cli.rb

```diff
@@ -18,6 +18,12 @@ class ChainloopCli < Formula
 
   depends_on "go" => :build
 
+  # build patch for nil pointer check, upstream pr ref, https://github.com/chainloop-dev/chainloop/pull/810
+  patch do
```

Comment:
We are releasing as we speak 0.88.1 so this patch should not be needed

Reply:
I'll keep you posted

Reply:
nice!!